TicTacToe/MoveManager.fs

```diff
@@ -12,11 +13,13 @@ let rec humanMove (input,game, enterMove : unit -> string, print : string -> uni
     else
         humanMove (input,game,enterMove,print)
 
-let computerMove (gameState : Game, isMinimax) : int =
-    if isMinimax || not(gameState.BoardSize = 3 && gameState.BoardSize = 4)  then
+let computerMove (gameState : Game, algorithmSeed) : int =
```

Comment:
I'm assuming you haven't worked on the story yet to make this algorithm selection more dynamic?  This function is getting pretty hairy...